dist/streams.js

```diff
@@ -32,12 +29,20 @@ class MetacomChunk {
     }
     throw new Error('Invalid chunk metadata: ' + metadata);
   }
+
+  static merge(ch1, ch2) {
+    const result = new Uint8Array(ch1.length + ch2.length);
+    result.set(ch1);
+    result.set(ch2, ch1.length);
+    return result;
+  }
```

Comment:
It's better to move this method out of class as a pure function

Reply:
Done